Hacker0x01/hai-on-hackerone
# replace bcolors with termcolor

## Issue number and link

Closes: https://github.com/Hacker0x01/hai-on-hackerone/issues/6

### 📝 Summary

This merge request is linked to issue #6. The main change is updating the print statement for colored console output. The previous method using `bcolors` has been replaced with the `colored` function for better readability and maintainability.

### 🧪 Test Plan

1. Run the script to verify that the output "Response from Hai" is displayed in light blue.
2. Ensure that there are no errors related to the `colored` function.
3. Check the terminal to confirm the color change from the old method to the new method.

### 🤔 Additional remarks

Switching to the `colored` function simplifies the code and improves consistency in how colors are applied across the project.

### ✅ Review Checklist

* [ ] Linked to the relevant Github issue
* [ ] MR status changed from draft to ready for review
* [ ] Comprehensive summary of the changes is written
* [ ] I ensured that unit, integration, regression, and end-to-end tests are added (if applicable)
* [ ] I assessed the performance impact of the changes (if applicable)
* [ ] I conducted a thorough self-review of the code on Github
* [ ] I ensured that all documentation is updated to reflect the changes
